### PR TITLE
Update release checklist

### PR DESCRIPTION
**Pull request by mikejolley**
Updates the release checklist with current processes. I will update this more as the week unfolds 😅

### CODE REVIEW

File: .github/release-initial-checklist.md [hunk 1]
```diff
@@ -27,26 +27,25 @@ The release pull request has been created! This checklist is a guide to follow f
...
--   [ ] Ask the porters of Rubik and Kirigami to test the built zip as well and to approve the PR if everything looks good.
-    -   [ ] Confirm all GitHub checks have passed on this PR prior to approving.
+
+## Smoke testing
 
 Each porter is responsible for testing the PRs that fall under the focus of their own team. Shared functionality should be tested by both porters. This means that the Rubik porter will mostly test checkout blocks and Store API endpoints, while the Kirigami porter will test the product related blocks and Store API endpoints.
 
+-   [ ] Smoke test the built release zip. Refer to the [Smoke testing checklist](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/internal-developers/testing/smoke-testing.md) for critical functionality.
+-   [ ] Test in a clean environment. Create a Jurassic.Ninja site, upload your zip, then smoke test it.
+-   [ ] Ask the porters of Rubik and Kirigami to test the built zip as well and to approve the PR if everything looks good. We recommend creating threads of p2 to track which test cases were tested.
```
Comment (alexflorisca) [inline .github/release-initial-checklist.md]: and Origami?

**Review by alexflorisca — APPROVED** [review]
I'll pre-approve this as I won't be here the rest of the week. Just left an inline comment about mentioning Origami as well.

**Comment by mikejolley** [comment]
@alexflorisca I updated both; can you look again please?

**Review by alexflorisca — APPROVED** [review]
Yep, all good 👍

**Pull request merged**